CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * [ENHANCEMENT] Add "integration" as a label for `cortex_alertmanager_notifications_total` and `cortex_alertmanager_notifications_failed_total` metrics. #3056
 * [ENHANCEMENT] Add `cortex_ruler_config_last_reload_successful` and `cortex_ruler_config_last_reload_successful_seconds` to check status of users rule manager. #3056
 * [ENHANCEMENT] Memcached dial() calls now have an optional circuit-breaker to avoid hammering a broken cache #3051
+* [ENHANCEMENT] `-ruler.evaluation-delay-duration` is now overridable as a per-tenant limit, `evaluation_delay_duration` #3098
```

Comment:
I think this would be considered a change.

Reply:
Yes, it's a YAML breaking change, because `evaluation_delay_duration` has been moved from `ruler_client` to `limits`. Technically, this is the kind of breaking changes we shouldn't introduce due to the 1.x guarantees we made.

Reply:
I've put the YAML field back, and as the flag hasn't changed and the flag behaviour is the same, I thought ENHANCEMENT would be more appropriate.  But not that fussed.

Reply:
If old YAML field is set, it still works (if new field is not set). We log a deprecation warning in such case. I've also added CHANGELOG entry about the rename.